### PR TITLE
Kork sqlize

### PR DESCRIPTION
**Pull request by dreynaud**
Some of the early SQL implementation in orca has been migrated and modernized independently in `kork-sql`. This PR makes orca's sql configuration a lot more like clouddriver's -- it should not change its behavior directly.

### CODE REVIEW

File: orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepository.java [hunk 1]
```diff
@@ -61,15 +64,18 @@ void resume(
   void updateStatus(ExecutionType type, @Nonnull String id, @Nonnull ExecutionStatus status);
 
   @Nonnull
+  @Instrumented(metricName = "retrieveById")
```
Comment (dreynaud) [inline orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepository.java]: small change here, this used to be named something not-very-descriptive like `retrieve2`. I favored more descriptive names

Comment (asher) [reply orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/ExecutionRepository.java]: :thankyou:

File: orca-queue-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlPendingExecutionConfiguration.kt [hunk 2]
```diff
@@ -13,7 +14,7 @@ import org.springframework.context.annotation.Configuration
 import java.time.Clock
 
 @Configuration
-@EnableConfigurationProperties(SqlProperties::class)
+@EnableConfigurationProperties(com.netflix.spinnaker.kork.sql.config.SqlProperties::class)
```
Comment (asher) [inline orca-queue-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlPendingExecutionConfiguration.kt]: It's imported so shouldn't need the package name here?

Comment (dreynaud) [reply orca-queue-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlPendingExecutionConfiguration.kt]: indeed, will remove

**Pull request merged**